e107_admin/update_routines.php

```diff
@@ -564,6 +566,21 @@ function update_core_database($type = '')
 }
 
 
+	function update_219_to_220($type='')
+	{
+		$sql = e107::getDb();
+		$just_check = ($type == 'do') ? false : true;
+
+		// add new pref contact_privacypolicy
+		if (!e107::getPref('contact_privacypolicy', ''))
+		{
+			e107::getConfig()->setPref('contact_privacypolicy', '')->save(false, true);
+		}
+
+		return $just_check;
+	}
+
+

```

Comment:
We don't actually need this, we just add the pref to default.xml and e107 will do the rest.